html/partials/nav.hbs

```diff
@@ -1,5 +1,5 @@
 <nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <a class="navbar-brand" href="/solve">Yellow Brick Code</a>
+  <a class="navbar-brand" href="/ybc">Yellow Brick Code</a>
```

Comment:
Is there any reason why we want to redirect user who clicked on the logo to a new page?
I think it might make more sense to simply redirect them to home page as that's how most webpages work.